snap/README.md

```diff
@@ -0,0 +1,172 @@
+# EdgeX UI Snap
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-ui)
+
+This folder contains snap packaging for the EdgeX UI Snap
+
+The project maintains a rolling release of the snap on the `edge` channel that is rebuilt and published at least once daily
+
+The snap currently supports the following architectures: `amd64`, `arm64`
+
+## Installation
+
+### Installing snapd
+
+The snap can be installed on any system that supports snaps. You can see how to install snaps on your system [here](https://snapcraft.io/docs/installing-snapd)
+
+However for full security confinement, the snap should be installed on an Ubuntu 18.04 LTS or later (Desktop or Server), or a system running Ubuntu Core 18 or later
+
+### Installing EdgeX UI as a snap
+The snap is published in the snap store at https://snapcraft.io/edgex-ui.
+You can see the current revisions available for your machine's architecture by running the command:
+
+```bash
+$ sudo snap info edgex-ui
+```
+
+The latest development version of the edgex-ui snap can be installed using:
+
+```bash
+$ sudo snap install edgex-ui --edge
+```
+
+**Note** - in general, installing from the edge channel is only recommended for development purposes
+
+**Note** - the snap has only been tested on Ubuntu Desktop
+
+### Install and configure edgexfoundry as a snap
+
+The latest development version of the edgexfoundry snap can be installed using:
```

Comment:
Add instructions for installing a stable version. Assume that the 2.1 channel exists.

Reply:
Added.